src/jdk.jcmd/share/man/jcmd.1

```diff
@@ -970,8 +969,9 @@ Impact: Medium --- pause time depends on number of loaded classes
 \f[I]subcmd\f[R]: must be either \f[V]static_dump\f[R] or
 \f[V]dynamic_dump\f[R] (STRING, no default value)
 .IP \[bu] 2
-\f[I]filename\f[R]: (Optional) Name of the shared archive to be dumped
-(STRING, java_pid<pid>_<subcmd>.jsa)
+\f[I]filename\f[R]: (Optional) Name of the shared archive to be dumped.
+If %p is specified in the filename, it is expanded to the JVM\[aq]s PID.
+(FILE, \[dq]java_pid%p_<subcmd>.jsa\[dq])
```

Comment:
The paragraph below this goes into details on how the default path is generated even though it is implied by the above help. We don't do this extra explanation for other places where the filename has a default, so probably not needed here, but I suppose it doesn't hurt either.

Also, two paragraphs below is a discussions about absolute paths that also applies to all other uses of filename. If it is necessary here, then that implies it is necessary in the other locations too.

Reply:
Yes - both of those paragraphs are _not_very_ useful.

"If `filename` is not specified, a default file name is chosen using the pid..." - this explains how the default filename works.

" If `filename` is not specified as an absolute path, the archive file is created in a directory relative to.." -- Well yes, we are saying that is if it's not absolute, it's relative... Hmm... This does not add much value either.

The good thing is these are not in the live help output, so deleting them does not make the help any more inconsistent.  I think these can both be removed.